APE0.rst

```diff
@@ -0,0 +1,360 @@
+Coordination Committee must solicit community feedback for no less than two
+weeks.
+
+Making Decisions
+^^^^^^^^^^^^^^^^
+To use its powers, the Coordination Committee must first attempt to reach a
+consensus. If a consensus cannot be reached the Committee votes. Every
+Coordination Committee member must either vote or explicitly abstain. Members
+with a conflict of interest on a particular vote must abstain. The final
+decision regarding what constitutes a conflict of interest is determined by the
+Ombudsperson in accordance with the conflict of interest policy. Passing
+decisions require a strict majority of non-abstaining Coordination Committee
+members.
+
+Whenever possible, the Coordination Committee's deliberations shall be held in
+public in a manner that permits discussion and comments from the community.
+Final decisions made by the Coordination Committee will be published in a
+publicly visible location.
+
+Electing the Coordination Committee
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A Coordination Committee election is held annually to fill any vacant seats.
+Notice of the election must be posted via the Project’s standard communication
+channels (reaching all active s) at least one month before the election begins.
+The election consists of two phases, each lasting two weeks:
+
+* Phase 1, Nominations: Candidates will be nominated and confirm their interest
+  in serving. Candidates can be nominated by anyone, including themselves, and
+  do not have to be Voting Members to be nominated.
+* Phase 2, Voting: Each Voting Member can vote for zero or more of the
+  candidates, up to the number of candidates. Voting is performed anonymously.
+  Candidates are ranked by the total number of votes they receive. If a tie
+  occurs, it may be resolved by mutual agreement among the candidates, or else
+  the winner will be chosen at random.
+
+The election process is managed by a non-voting Returns Officer nominated by the
+outgoing Coordination Committee. For the initial election, the Returns Officer
+will be nominated by the NumFOCUS Executive officer or their delegate. Elections
+will be carried out every year, provided that at least one of the Coordination
+Committee members has reached the end of their term or decided to step down.
+
+Term
+^^^^
+Each Coordination Committee member's term runs for three years from when
+the election results are finalized, except when the election was to fill a
+vacancy created partway through a term. In that case, the newly-elected member’s
+term runs for the remainder of the term of the person they are replacing. There
+is no limit to the number of terms that a single individual can be elected for.
+
+For the initial election of Coordination Committee members, two seats will have
+terms of one year, one has a term of two years, and one has a term of three
+years to create a staggered set of replacements and provide continuity in the
+Coordination Committee.
```

Comment:
I have one other item I want to bring up as a suggested change, which I'm giving here as replacement wording for this particular paragraph:

```suggestion
For the initial elections of Coordination Committee members, the existing
Coordination Committee seats will come up for election in a staggered manner.
The longest-serving two committee members will come up for re-election in the
first election, the third-longest in the next year, and the shortest-serving in
the third year.  This provides continuity in the Coordination Committee while
transitioning to the staggered election model outlined in this Charter.
```
My argument for this is that it ensures that the first election does not feature a huge turnover and loss of continuity, as this is exactly what the "staggered" elections are meant to avoid.  Without this change it's possible that *no* existing CoCo members would carry over (either by choice or by vote), which would be highly problematic given the amount of institutional knowledge and unwritten tasks that currently exist in the CoCo.

(Also, while it may look like a conflict-of-interest given I'm a CoCo member, with my proposed suggestion it's not, as I'm one of the two longest-serving members, so either way I'm up in the same election.)

I know that's a bit odd to suggest this since the PR is mine, but in this case it's something that came up after the Working Group concluded (indeed, it's partly that I'm not sure the current wording reflects the *last* meeting of the Working Group), so I didn't want to force it upon the document without others having a chance to weigh in.

Reply:
I find it very problematic to call it a rework of governance and the status quo when members of the new committee are hard wired basically for status quo before any voting can happen. There is already overlap ensured by current coco members serving on the finance committee, etc.
Chances that none of the current coco members would pass the voting is slim, but in case it happens that's also a valuable feedback from the community.

Reply:
> very problematic to call it a rework of governance 

To be clear: up until this point I don't think anyone called it a "rework of governance" - rather the goal most of the WG was thinking about was writing down what's currently true, and then improving it where writing it down makes us realize improvements are needed.  I'd say that's fully compatible with a ~50% turn-over.

Put another way: the goal had never been said to be a complete change of the entire philosophy of Project governance.  So my proposal here is aimed at helping there be incremental change with some continuity that allows a complete transformation on a 3-year timescale if needed.

Reply:
Oh and since it's not otherwise in this document: also the more "rework" elements are likely to happen in follow-on discussions about policy and the like. The CoCo's *original* thinking was that that's where we wanted to have some of the more dramatic re-working, by sub-groups that actually have the expertise and interest to do so.  But the point of this Charter is  that the WG felt otherwise and that there was a desire for the "foundation" to be set before any of that could be done.

Reply:
Sorry, should have responded to the whole thing - 

> Chances that none of the current coco members would pass the voting is slim, but in case it happens that's also a valuable feedback from the community.

I'm actually more concerned that a sizable chunk of the existing coco would *choose* not to go up for re-election. I could see scenarios where I don't depending on timing, other interested parties, etc. So I'm making this suggestion to ensure this doesn't happen all-at-once because I think with no or just a single-member overlap there would be a disastrous dropping-of-balls given the current spread of duties in the CoCo.

Reply:
This hard-wiring is still extreme worrisome. The same pushback for any potential change from the current status quo as was experienced during the WG process.

Reply:
I agree that this should not be hard-wired. Personally, I think some continuity is important. BUT - that should be decided by the community through elections and not by a small group making edits to this PR. If the community wants continuity, then they will vote for (some of) the current Coco members. If the community instead wants a "fresh start", they are going to vote for new people. So, don't put any restrictions into this document. Leave it up to the vote.

Reply:
"Fresh start" might sound nice, but there is also a risk of fresh people making the same mistakes again because they lost the "lessons learned" from existing CoCo. There needs to be a balance between nothing ever changing and changing too fast.

Reply:
so you suggest that if people wouldn't get the vote they in effect would disappear with all the "lessons learned" from the face of the Earth rather than helping the new committee, so it's better keep them hard wired?

Reply:
I don't know how this will play out, really. If people are voted out, they might or might not make themselves available. For the people who vote them out, do they want to consult those they voted out in the first place? 🤷

Reply:
Can we just have our cake and eat it? Write here that the hope/expectation is that some of the CoCo members will stand for the shorter-duration initial memberships?  That way we can have the vote and indeed let the community decide.

To the extent it matters, I am super thankful for those on the current committee and feel they've done a good job, even if some balls were dropped (would be far worse if I'd've been on it...). I certainly hope some will stand and I'll vote towards having some continuity.

Reply:
Thanks for this perspective, @hamogu and @bsipocz.  I'm swayed by your points that if most of the community really disagrees with the perspective of e.g. me, @pllim, and @mhvk that continuity is important, then that's the will of the community and that's the reality.

I still worry about the loss of continuity by *choice* of the CoCo, but as someone else pointed out, that can happen at any time really since CoCo members can resign, so this isn't a good mechanism for that.

I might offer a follow-up suggestion along the lines of @mhvk's comment to state that continuity is fine *if the community votes for it*, but I think for all elections other than the first one that's implied by the staggering process anyway.

So I'm going to withdraw my suggestion here (I'll mark it "resolved" when a round of edits happens and will plan not to include this unless there's others who pop in and say they *do* want it after all).

Reply:
I would be very surprised if none of the current CoCo get elected in the first round of elections. That being said, I think it would be very self-defeating to enforce continuity here.

Reply:
Since we are now in a new round of edits, I'll mark this as resolved without applying the suggestion as mentioned by @eteq in https://github.com/astropy/astropy-APEs/pull/61#discussion_r502558084